### PR TITLE
[CDAP-16359] Integrate with nextruntime batch API from backend for pipelines

### PR DESCRIPTION
**Pull request by ajainarayanan**
JIRA: https://issues.cask.co/browse/CDAP-16359
Build: https://builds.cask.co/browse/CDAP-UDUT522-1

Pending: Add E2E tests for pipelines list view.

Required backend changes available in latest develop (https://github.com/cdapio/cdap/pull/11899)

### CODE REVIEW

File: cdap-ui/graphql/helpers/BatchEndpoints/nextRuntime.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  options.body = body;
+
+  let nextRuntimeInfo = await requestPromiseWrapper(options, auth);
+
+  const runsMap = {};
+  nextRuntimeInfo.forEach((run) => {
+    if (run.schedules) {
+      runsMap[run.appId] = run.schedules;
+    } else {
+      runsMap[run.appId] = [];
```
Comment (elfenheart) [inline cdap-ui/graphql/helpers/BatchEndpoints/nextRuntime.js:40]: should this be empty object? I see in the schema, the response structure per schedule is 
```
{
  id,
  time
}
```

Comment (ajainarayanan) [reply cdap-ui/graphql/helpers/BatchEndpoints/nextRuntime.js:40]: the nextruntime is an array as defined [here](https://github.com/cdapio/cdap/pull/11912/files#diff-0bab0b762798a2c7bda6f7cea7cc72c2R28) and the schedules are inturn another array (an app can have more than one schedule)

**Review by elfenheart — APPROVED** [review]
👍 LGTM

**Comment by ajainarayanan** [comment]
Green build: https://builds.cask.co/browse/CDAP-UDUT522-2

**Pull request merged**